packages/contracts-bedrock/test/L2/GasPriceOracle.t.sol

```diff
@@ -195,4 +203,114 @@ contract GasPriceOracleEcotone_Test is GasPriceOracle_Test {
         // gas * (2M*16*20 + 3M*15) / 16M == 48977.5
         assertEq(price, 48977);
     }
+
+    /// @dev Tests that `setFjord` is only callable by the depositor.
+    function test_setFjord_wrongCaller_reverts() external {
+        vm.expectRevert("GasPriceOracle: only the depositor account can set isFjord flag");
+        gasPriceOracle.setFjord();
+    }
+}
+
+contract GasPriceOracleFjordActive_Test is GasPriceOracle_Test {
```

Comment:
Aren't we missing a test of `getL1FeeUpperBound`? We could add them to the last 2 tests.